packages/field-base/test/checked-mixin.test.js

```diff
@@ -0,0 +1,97 @@
+import { expect } from '@esm-bundle/chai';
+import { fixtureSync } from '@vaadin/testing-helpers';
+import { PolymerElement, html } from '@polymer/polymer/polymer-element.js';
+import { CheckedMixin } from '../src/checked-mixin.js';
+import { InputSlotMixin } from '../src/input-slot-mixin.js';
+
+customElements.define(
+  'checked-mixin-element',
+  class extends CheckedMixin(InputSlotMixin(PolymerElement)) {
+    static get template() {
+      return html`<div>
+        <slot></slot>
+        <slot name="input"></slot>
+      </div>`;
+    }
+
+    constructor() {
+      super();
+
+      this._setType('checkbox');
+    }
+  }
+);
+
+describe('checked-mixin', () => {
+  let element, input, link;
+
+  describe('delegation', () => {
+    describe('checked attribute', () => {
+      beforeEach(() => {
+        element = fixtureSync(`<checked-mixin-element checked></checked-mixin-element>`);
+        input = element.querySelector('[slot=input]');
+      });
+
+      it('should delegate checked attribute to the input', () => {
+        expect(input.hasAttribute('checked')).to.be.true;
+
+        element.removeAttribute('checked');
+        expect(input.hasAttribute('checked')).to.be.false;
+      });
+    });
+  });
+
+  describe('default', () => {
```

Comment:
nit: typically `default` or `basic` suite comes first in most of our tests, could you update this one please?

Reply:
Good point! Done.